gluecodium/src/test/resources/smoke/inheritance/output/dart/lib/src/_type_repository.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:library/src/BuiltInTypes__conversion.dart';
+import 'package:library/src/_type_repository.dart' as __lib;
+import 'package:library/src/smoke/ChildClassFromClass.dart';
+import 'package:library/src/smoke/ChildClassFromInterface.dart';
+import 'package:library/src/smoke/ChildInterface.dart';
+import 'package:library/src/smoke/InternalChild.dart';
+import 'package:library/src/smoke/InternalParent.dart';
+import 'package:library/src/smoke/ParentClass.dart';
+import 'package:library/src/smoke/ParentInterface.dart';
+import 'dart:ffi';
+import 'package:ffi/ffi.dart';
+final Map<String, Function> typeRepository = {
```

Comment:
HashMap?

Reply:
Map<> here is just a hint for the type resolution to help it disambiguate {} to a map (not a set) when/if it's empty. Real implementation is created by {} on the right side of assignment. According to Dart sources, it's LinkedHashMap by default.